src/org/labkey/test/components/ui/navigation/UserMenu.java

```diff
@@ -45,4 +45,14 @@ protected Locator getToggleLocator()
     {
         return Locator.id(MENU_ID);
     }
+
+    /**
+     * The button for the drop down user menu. The user menu on the SampleManager app is an example of this button.
+     *
+     * @return A locator to the drop down user menu button.
+     */
+    public static Locator AppUserMenu()
```

Comment:
Java method names shouldn't be capitalized.

Reply:
Yep, linter said the same thing. Fixed.